openfecwebapp/templates/macros/filters/date.html

```diff
@@ -1,31 +1,23 @@
 {% macro field(name, label, dates) %}
-
   <div class="filter js-filter" id="{{ name }}" data-name="{{ name }}" data-filter="date">
   <fieldset>
     <legend for="{{ name }}" class="label">{{ label }}</legend>
-    <ul>
-      <li>
-        <input id="radio-1" name="_{{ name }}" type="radio" data-min-date={{ dates['year'][0] | date(fmt='%m-%d-%Y') }} data-max-date={{ dates['year'][1] | date(fmt='%m-%d-%Y') }}>
-        <label for="radio-1">This year</label>
-      </li>
-      <li>
-        <input id="radio-2" name="_{{ name }}" type="radio" data-min-date={{ dates['cycle'][0] | date(fmt='%m-%d-%Y') }} data-max-date={{ dates['cycle'][1] | date(fmt='%m-%d-%Y') }}>
-        <label for="radio-2">This cycle</label>
-      </li>
-      <li>
-        <input id="radio-3" name="_{{ name }}" type="radio">
-        <label for="radio-3">Custom</label>
-      </li>
-    </ul>
-    <div class="date-range-input">
-      <label for="min_{{ name }}">From</label>
-      <input type="text" id="min_{{ name }}" name="min_{{ name }}" class="js-min-date" data-prefix="Beginning" data-inputmask="'alias': 'mm/dd/yyyy'">
-      <label for="max_{{ name }}">To</label>
-      <input type="text" id="max_{{ name }}" name="max_{{ name }}" class="js-max-date" data-prefix="Ending" data-inputmask="'alias': 'mm/dd/yyyy'">
+    <div class="range range--date js-date-range">
```

Comment:
Do you think it'd be worth abstracting this so that it can be shared by the other date filter below? It might be unnecessary, but just a thought.